README.md

```diff
@@ -69,9 +69,9 @@ This project is a follow-up to the now unmaintained (and closed-source) [Message
 - **Swipe left** to cycle between selected layouts (languages).
 - **Swipe to bottom-left** to toggle voice input. Requires [FUTO Voice Input](https://play.google.com/store/apps/details?id=org.futo.voiceinput).
 
-### Symbol (`#`) Key
+### Symbol/Letters (`#`/`ABC`) Key
```

Comment:
Seems a bit too verbose now, lets just make this : 

`### Symbols / Letters Key`